django/db/models.py

```diff
@@ -349,16 +346,16 @@ def __unicode__(self):
 class SmallMolecule(models.Model):
     
     nominal_target          = models.ForeignKey('Protein', null=True)
-    facility_id             = _CHAR(max_length=_FACILITY_ID_LENGTH, **_NOTNULLSTR)
-    salt_id                 = _CHAR(max_length=_SALT_ID_LENGTH, **_NOTNULLSTR)
-    lincs_id                = _CHAR(max_length=15, **_NULLOKSTR)
+    facility_id             = _TEXT(max_length=_FACILITY_ID_LENGTH, **_NOTNULLSTR)
```

Comment:
I'll fix this - looks like an oversight on my part.  Note that max_length has no effect on our system as we aren't using the Django model forms https://docs.djangoproject.com/en/1.8/ref/models/fields/#textfield.